Kuadrant/multicluster-gateway-controller
# #472: Add Interval field to DNS Health checks

> Closes #472

## Description

Add spec.healthCheck.interval field that allows the customisation of the interval the health checks are performed at

## Verification steps

1. Run the walkthrough until the step of the creation of the DNSPolicy
2. Create the DNSPolicy with the following field in the step:
    ```yaml
    healthCheck:
      interval: 90s
    ```
3. Verify that the resulting DNSHealthCheckProbes have the interval set to the interval from the DNSPolicy
